openfisca_aotearoa/parameters/entitlements/social_security/accommodation_supplement/age_threshold.yaml

```diff
@@ -0,0 +1,5 @@
+description: Minimum age to recieve accommodation supplement
+reference: TODO
```

Comment:
I would recommend not adding a `reference` at all, or leaving it empty.
As free text is allowed, this will be parsed and exposed to the users. It will also make it harder to automatically detect missing references.